src/Tizen.Multimedia.MediaPlayer/Player/PlayerError.cs

```diff
@@ -49,6 +49,7 @@ internal enum PlayerErrorCode
         NotSupportedAudioCodec = PlayerErrorClass | 0x0e,
         NotSupportedVideoCodec = PlayerErrorClass | 0x0f,
         NotSupportedSubtitle = PlayerErrorClass | 0x10,
+        NotSupportedFormat = PlayerErrorClass | 0x11,
```

Comment:
Please add `NotSupportedFormat` into PlayerErrorCodeExtensions.GetException() method.

Reply:
I'll fix it.